web/views/api.py

```diff
@@ -23,11 +23,8 @@
 from core.models import (
```

Comment:
At the very top of this file, the `import os` is not needed it seems

Reply:
removed, thanks